families/seth/rpc/src/block.rs

```diff
@@ -16,13 +16,79 @@
  */
 
 use jsonrpc_core::{Params, Value, Error};
+use protobuf;
+use uuid;
 
-use sawtooth_sdk::messaging::zmq_stream::ZmqMessageSender;
-use super::error;
+use error;
+
+use sawtooth_sdk::messaging::zmq_stream::*;
+use sawtooth_sdk::messaging::stream::*;
+
+use sawtooth_sdk::messages::client::{
+    ClientBlockListRequest, ClientBlockListResponse, PagingControls,
+};
+use sawtooth_sdk::messages::block::BlockHeader;
+use sawtooth_sdk::messages::validator::Message_MessageType;
 
 pub fn block_number(_params: Params, mut sender: ZmqMessageSender) -> Result<Value, Error> {
```

Comment:
Could you add a quite comment on what this does?  Is it "get a block by block number"?